### PR TITLE
Elastic CI Stack docs unification

### PR DESCRIPTION
**Pull request by karensawrey**

### CODE REVIEW

File: app/views/layouts/_sidebar.html.erb [hunk 1]
```diff
@@ -40,10 +40,12 @@
         </ul>
         <p class="Docs__nav__section-subheading">Elastic CI Stack for AWS</p>
         <ul class="Docs__nav__sub-nav">
-          <%= sidebar_link_to "Running Elastic CI Stack", 'agent/v3/elastic-ci-aws' %>
-          <%= sidebar_link_to "Template Parameters", 'agent/v3/elastic-ci-aws/parameters' %>
+          <%= sidebar_link_to "Overview", 'agent/v3/elastic-ci-aws/elastic-ci-stack-overview' %>
+          <%= sidebar_link_to "Elastic CI Stack for Linux", 'agent/v3/elastic-ci-aws' %>
```
Comment (plaindocs) [inline app/views/layouts/_sidebar.html.erb:44]: This isn't just Linux, it's either just Elastic CI Stack, or it's Linux and Windows. Not quite sure which is the least worst option here.

File: pages/agent/v3/elastic_ci_aws.md.erb [hunk 2]
```diff
@@ -82,7 +77,7 @@ You're not required to create any special IAM roles or policies, though the depl
 
 ### Key creation
 
-It’s not required to create keys for the default deployment of Elastic CI Stack, but you can additionally create:
+It's not required to create keys for the default deployment of Elastic CI Stack, but you can additionally create:
```
Comment (plaindocs) [inline pages/agent/v3/elastic_ci_aws.md.erb:80]: Let's be more direct here. "You don't need to"...

File: pages/agent/v3/elastic_ci_aws.md.erb [hunk 3]
```diff
@@ -1,26 +1,21 @@
...
 <a href="https://console.aws.amazon.com/cloudformation/home#/stacks/new?stackName=buildkite&templateURL=https://s3.amazonaws.com/buildkite-aws-stack/latest/aws-stack.yml"><%= image "launch-stack.svg", alt: "Launch stack button" %></a>
 
 <!-- vale on -->
 
-> The stack creates its own VPC (virtual private cloud) by default. Best practice involves setting up a separate development AWS account and using role switching and consolidated billing. Refer to the [Delegate Access Across AWS Accounts tutorial](http://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html) for more information.
+{:toc}
+
+## Before you start
+
+> Elastic Stack for AWS creates its own VPC (virtual private cloud) by default. It's recommended to set up a separate development AWS account and use role switching and consolidated billing. See the tutorial on how to ["Delegate Access Across AWS Accounts"](http://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html) for more information.
```
Comment (plaindocs) [inline pages/agent/v3/elastic_ci_aws.md.erb:18]: Shall we mention that this is an external AWS tutorial so folks know they are leaving buildkite.com /docs

File: pages/agent/v3/elastic_ci_aws/elastic_ci_stack_overview.md.erb [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+[Buildkite agent](/docs/agent/v3) cluster.
+
+You can use the Buildkite Elastic CI Stack to parallelize large test
+suites across hundreds of nodes, run tests, app deployments, or AWS ops tasks.
+
+Each Buildkite Elastic CI Stack deployment contains an Auto Scaling group and a launch template.
+
+Get started with Buildkite Elastic CI Stack for:
+
+* [Linux (and Windows)](/docs/agent/v3/elastic-ci-aws)
```
Comment (plaindocs) [inline pages/agent/v3/elastic_ci_aws/elastic_ci_stack_overview.md.erb:13]:
```suggestion
* [Linux and Windows](/docs/agent/v3/elastic-ci-aws)
```

File: pages/agent/v3/elastic_ci_aws/parameters.md.erb [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# Elastic CI Stack for AWS Parameters
+# Elastic CI Stack for AWS parameters
 
 The following tables list all the available template parameters. These can be
```
Comment (plaindocs) [inline pages/agent/v3/elastic_ci_aws/parameters.md.erb:3]: This definitely needs a little more intro. What template from where. 

Something like if you're using the Cloudformation template (from ...) to create an Auto Scaling group, launch template, and host resource group for maintaining a pool of EC2 instances you can use these parameters to create an evironment to match your exact needs

@keithduncan does this sound about right?

**Review by plaindocs — CHANGES_REQUESTED** [review]
This is already looking much better.

Can we match up the page titles and menu titles a bit more? If I click on **Customizing Elastic CI Stack** and end up on **Elastic CI Stack for AWS parameters** I'm gonna wonder what happened.

It also looks like we should linking the Cloun Formation Role more closely to the template parameters, unless I'm getting confused about two different templates, in which case even more work is needed.

File: app/views/layouts/_sidebar.html.erb (hunk 1 above)
Comment (karensawrey) [reply app/views/layouts/_sidebar.html.erb:44]: My thoughts exactly :D "Not quite sure which is the least worst option"
I'll go with "just Elastic CI Stack".

File: pages/agent/v3/elastic_ci_aws.md.erb (hunk 3 above)
Comment (karensawrey) [reply pages/agent/v3/elastic_ci_aws.md.erb:18]: Yep!

File: pages/agent/v3/elastic_ci_aws/parameters.md.erb (hunk 5 above)
Comment (keithduncan) [reply pages/agent/v3/elastic_ci_aws/parameters.md.erb:3]: > What template from where.

These parameters are for the [aws-stack.yml](https://github.com/buildkite/elastic-ci-stack-for-aws/blob/master/templates/aws-stack.yml) template which we link to the deployed and versioned copies of on S3.

> and host resource group

This template doesn’t create a host resource group, that’s an Elastic Mac template thing 😄 

> you can use these parameters to create an evironment to match your exact needs

Maybe swap environment for something like "use these parameters to configure the EC2 instances to suit your needs"?

**Comment by keithduncan** [comment]
> It also looks like we should linking the Cloun Formation Role more closely to the template parameters, unless I'm getting confused about two different templates, in which case even more work is needed.

The main "Elastic CI Stack for AWS" template and "CloudFormation Service Role" templates are distinct. The service role template is wholly optional, and has a limited audience, where as the former main template is what everyone uses.

**Comment by karensawrey** [comment]
Thank you so much for the review, @keithduncan!

File: pages/agent/v3/elastic_ci_aws.md.erb [hunk 6]
```diff
@@ -1,26 +1,21 @@
...
 <a href="https://console.aws.amazon.com/cloudformation/home#/stacks/new?stackName=buildkite&templateURL=https://s3.amazonaws.com/buildkite-aws-stack/latest/aws-stack.yml"><%= image "launch-stack.svg", alt: "Launch stack button" %></a>
 
 <!-- vale on -->
 
-> The stack creates its own VPC (virtual private cloud) by default. Best practice involves setting up a separate development AWS account and using role switching and consolidated billing. Refer to the [Delegate Access Across AWS Accounts tutorial](http://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html) for more information.
+{:toc}
+
+## Before you start
+
+> Elastic Stack for AWS creates its own VPC (virtual private cloud) by default. It's recommended to set up a separate development AWS account and use role switching and consolidated billing. You can check out this external tutorial for more information on how to ["Delegate Access Across AWS Accounts"](http://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html).
```
Comment (plaindocs) [inline pages/agent/v3/elastic_ci_aws.md.erb:18]: I'd quite like to rephrase this "it's recommended" as well. Maybe just "Best practice is to set up..."

File: pages/agent/v3/elastic_ci_aws/parameters.md.erb [hunk 7]
```diff
@@ -1,7 +1,6 @@
-# Elastic CI Stack for AWS Parameters
+# Elastic CI Stack for AWS parameters
```
Comment (plaindocs) [inline pages/agent/v3/elastic_ci_aws/parameters.md.erb:1]: H1 is Parameters, no?

**Review by plaindocs — APPROVED** [review]
I've made a few last suggestions, but this is a useful improvement, looking forward to merging in January.

File: pages/agent/v3/elastic_ci_aws.md.erb [hunk 8]
```diff
@@ -1,26 +1,21 @@
...
 <a href="https://console.aws.amazon.com/cloudformation/home#/stacks/new?stackName=buildkite&templateURL=https://s3.amazonaws.com/buildkite-aws-stack/latest/aws-stack.yml"><%= image "launch-stack.svg", alt: "Launch stack button" %></a>
 
 <!-- vale on -->
 
-> The stack creates its own VPC (virtual private cloud) by default. Best practice involves setting up a separate development AWS account and using role switching and consolidated billing. Refer to the [Delegate Access Across AWS Accounts tutorial](http://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html) for more information.
+{:toc}
+
+## Before you start
+
+> Elastic Stack for AWS creates its own VPC (virtual private cloud) by default. THe best practice is to set up a separate development AWS account and use role switching and consolidated billing. You can check out this external tutorial for more information on how to ["Delegate Access Across AWS Accounts"](http://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html).
```
Comment (plaindocs) [inline pages/agent/v3/elastic_ci_aws.md.erb:18]:
```suggestion
> Elastic Stack for AWS creates its own VPC (virtual private cloud) by default. Best practice is to set up a separate development AWS account and use role switching and consolidated billing. You can check out this external tutorial for more information on how to ["Delegate Access Across AWS Accounts"](http://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html).
```

**Comment by karensawrey** [comment]
Let's merge? @plaindocs

**Pull request merged**